app/data_model/database.py

```diff
@@ -32,15 +32,17 @@ class EQSession(base):
     __tablename__ = 'eq_session'
     eq_session_id = Column('eq_session_id', String, primary_key=True)
     user_id = Column('user_id', String, primary_key=True)
-    timestamp = Column('timestamp', DateTime)
+    session_data = Column('session_data', String)
+    created_at = Column('created_at', DateTime, default=datetime.datetime.utcnow)
+    updated_at = Column('updated_at', DateTime, default=datetime.datetime.utcnow, onupdate=datetime.datetime.utcnow)
 
-    def __init__(self, eq_session_id, user_id):
+    def __init__(self, eq_session_id, user_id, session_data):
         self.eq_session_id = eq_session_id
         self.user_id = user_id
-        self.timestamp = datetime.datetime.now()
+        self.session_data = session_data
 
     def __repr__(self):
-        return "<EQSession('%s', '%s', '%s')>" % (self.eq_session_id, self.user_id, self.timestamp)
+        return "<EQSession('%s', '%s', '%s')>" % (self.eq_session_id, self.user_id, self.session_data)
```

Comment:
Is it valuable to have the data in this?

Reply:
Guess not. Will remove.